Update Troubleshooting Google Ads Conversions section (#4225)
segmentio/segment-docs: src/connections/destinations/catalog/google-ads-gtag/index.md

      ```

2. Verify that the [Google Conversion ID](/docs/connections/destinations/catalog/google-ads-gtag/#google-conversion-id) in your Segment workspace is correct.

> 3. Find your ad online and **click on it**. This will redirect you to your website.

Reviewer: ```suggestion
3. Find your ad online and click it. This will redirect you to your website.
```

8. See if the value for the `ct_cookie_present` changed to `true`. If `true`, it means that Google Ads counts the event as a conversion.  

> info ""

> > Google Ads will only consider an event as a conversion when the user arrived at your website coming from an Ad _click_. The Google SDK is responsible for checking if the user came from an Ad click and sets the parameter `ct_cookie_present` to true. Without clicking through an ad, Google Ads won't reflect the conversion because this information is missing in the network requests.

Reviewer: ```suggestion
> Google Ads considers an event as a conversion when the user arrived at your website as a result of an Ad _click_. The Google SDK is responsible for checking if the user came from an Ad click and sets the parameter `ct_cookie_present` to true. Without clicking through an ad, Google Ads doesn't reflect the conversion because this information is missing in the network requests.
```

Present tense
